stop_and_wait_plugin/src/stop_and_wait_plugin.cpp

```diff
@@ -199,8 +201,9 @@ namespace stop_and_wait_plugin
             }
             else
             {
-                double jerk_req = (2*start_speed)/pow(maneuver_time_,2);
-                //double start_speed = current_speed_;    //Get static value of current speed at start of planning
+                //double jerk_req = (2*start_speed)/pow(maneuver_time_,2);
+                double jerk_req = 2.0; //Setting constant for testing:vanden-plas
```

Comment:
Is this hardcoded Jerk expected?

Reply:
If so shouldn't it be a parameter?

Reply:
This shouldn't be hardcoded, but currently making it dynamic is causing the vehicle to prompt a route_completed message without completely bringing it to a stop.

Reply:
Ok we will leave it for later per your other investigation